prow/jobs/pingcap/tidb/tidb-postsubmits.yaml

```diff
@@ -0,0 +1,9 @@
+postsubmits:
+  pingcap/tidb:
+    - name: pingcap/tidb/merged_e2e_test
+      agent: jenkins
+      decorate: false # need add this.
+      context: wip-e2e-test
+      skip_report: true
```

Comment:
when the job ready, please change those lines

you can view the jobs at https://prow.tidb.net/?repo=pingcap%2Ftidb&type=postsubmit